### DISCUSSION TITLE
The output file log is too long, I want to hide it and only keep the build time

### DISCUSSION THREAD

**Discussion by wengjiacheng** [web-infra-dev/rsbuild#3451] [Q&A] [answered]
打包输出的文件日志太长，想隐去，只保留打包时长
rspack: (config, { appendPlugins }) => {
      config.stats = {}
      config.outputPath = false
      return config
    },
    设置了这个没生效

**Comment by chenjiahan** [comment, marked as answer]
Use this config: https://rsbuild.dev/config/performance/print-file-size
